core/storage/opportunity/gae_models_test.py

```diff
@@ -59,20 +65,6 @@ def setUp(self):
             assigned_voice_artist_in_language_codes=[]
         ).put()
 
-    def test_get_deletion_policy(self):
-        self.assertEqual(
-            opportunity_models.ExplorationOpportunitySummaryModel
-            .get_deletion_policy(),
-            base_models.DELETION_POLICY.KEEP_IF_PUBLIC)
-
-    def test_has_reference_to_user_id(self):
```

Comment:
I thought we were only going to delete these methods for DELETION_POLICY.NOT_APPLICABLE, and keep them otherwise?

I.e. there still needs to be a test for whether the relevant method returns true/false?

Reply:
Returned back.